dbscripts/xml/upgrade/3.0.0_reviewer_files.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * 3.0.0_reviewer_files.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * 3.0.0.0 reviewer files update fix.
+  -->
+<data>
+	<sql>
+		<!-- #2173 the files should have file_stage = 5 (SUBMISSION_FILE_REVIEW_ATTACHMENT), assoc_type should be 517 (ASSOC_TYPE_REVIEW_ASSIGNMENT), assoc_id should be the appropriate review_id. -->
+		<query driver="mysql">UPDATE submission_files sf, review_assignments ra SET sf.file_stage = 5, sf.assoc_type = 517, sf.assoc_id = ra.review_id WHERE ra.reviewer_file_id IS NOT NULL AND sf.file_id = ra.reviewer_file_id AND sf.revision = 1</query>
+		<query driver="mysqli">UPDATE submission_files sf, review_assignments ra SET sf.file_stage = 5, sf.assoc_type = 517, sf.assoc_id = ra.review_id WHERE ra.reviewer_file_id IS NOT NULL AND sf.file_id = ra.reviewer_file_id AND sf.revision = 1</query>
+		<query driver="postgres7">UPDATE submission_files SET file_stage = 5, assoc_type = 517, assoc_id = ra.review_id FROM review_assignments ra WHERE ra.reviewer_file_id IS NOT NULL AND submission_files.file_id = ra.reviewer_file_id AND submission_files.revision = 1</query>
+		<!-- #2173 insert a raw in review_round_files -->
+		<query>INSERT INTO review_round_files (submission_id, review_round_id, stage_id, file_id, revision) SELECT ra.submission_id, ra.review_round_id, 3, ra.reviewer_file_id, 1 FROM review_assignments ra WHERE ra.reviewer_file_id IS NOT NULL</query>
```

Comment:
Mind adding a comment with the name of the constant for magic numbers, e.g. 3 in this query?